x-pack/plugins/security_solution/common/constants.ts

```diff
@@ -170,6 +171,7 @@ export const USERS_PATH = '/users' as const;
 export const KUBERNETES_PATH = '/kubernetes' as const;
 export const NETWORK_PATH = '/network' as const;
 export const MANAGEMENT_PATH = '/administration' as const;
+export const COVERAGE_OVERVIEW_PATH = '/coverage_overview' as const;
```

Comment:
Should we make the page URL more specific? `rules_coverage_overview`, `detections_coverage_overview`, `rules/coverage_overview`, etc?
@approksiu what URL are we going to use for the page?

Reply:
rules_coverage_overview is fine with me.